src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -2416,7 +2416,7 @@ static void M()
 
         // TODO: extra error CS1014
         [Fact]
-        public void CS1043ERR_SemiOrLBraceExpected()
+        public void CS7887ERR_SemiOrLBraceOrArrowExpected()
```

Comment:
Yes, but we can probably retire the old code and uniformly use the new error code.

---
In reply to: [77885635](https://github.com/dotnet/roslyn/pull/13543#discussion_r77885635) [](ancestors = 77885635)